DuckDuckGo/AppDelegate.swift

```diff
@@ -66,6 +67,10 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         }
         #endif
 
+        CrashCollection.run { params in
+            Pixel.fire(pixel: .dbCrashDetected, withAdditionalParameters: params)
+        }   
```

Comment:
Looks like there's a SwiftLint warning here to clean up

```suggestion
        }
```